### DISCUSSION TITLE
Another release: 0.28.0 breaking change in topic names

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/WeConnect-mqtt#73] [Announcements]
I want to apologise for the second breaking name change in the topic naming in two days, but publishing and subscribing to the same topics caused too many difficulties and errors. Thus I needed to drop those. So if you want to write to a topic now you need to add `_writetopic` to the topic you want to change.
I decided to do this second change quick in the hope that not too many of you already changed the topics for 0.27.0 release.
For the users of FHEM I added a wiki page [here](https://github.com/tillsteinbach/WeConnect-mqtt/wiki/FHEM-examples) feel free to add/improve. If you use another home automatization software feel free to add another page to the wiki.

### Changed
- All writable items have now two topics: topicname and topicname_writetopic ***Warning this is a breaking change in the topic naming, you have to change your subscriptions!***
  ***Sorry! I was fed up with the way MQTT behaves when you subscribe to the same topic you publish to.***
- Updated API to 0.35.0

### Fixed
- The changes fix several problems when messages are received that were published by WeConnect-MQTT itself

<hr /><em>This discussion was created from the release <a href='https://github.com/tillsteinbach/WeConnect-mqtt/releases/tag/v0.28.0'>0.28.0</a>.</em>

**Comment by diplix** [comment]
good choice, i was wondering why command and state topics were not seperated. now they are and everything seems to work smoothly.

**Reply by tillsteinbach** [reply]
Yes, it was a bad idea. I thought it is easier that way for software that autocreates states, but I think now it is a better compromise with the topic that contains all available topics. I was not experienced with MQTT before.
